discovery-provider/tests/utils.py

```diff
@@ -211,7 +212,7 @@ def populate_mock_db(db, entities, block_offset=0):
 
         for i, aggregate_play_meta in enumerate(aggregate_plays):
             aggregate_play = models.AggregatePlays(
-                play_item_id=aggregate_play_meta.get("play_item_id", i),
+                play_item_id=aggregate_play_meta.get("play_item_id", i+1),
```

Comment:
is this to prevent play id = 0? (let's add a comment if so)

Reply:
but it looks like everything including tracks starts at 0 so i'll just change it back to that haha